proxy-node-shared/src/test/java/uk/gov/ida/notification/shared/IstioHeaderMapperFilterTest.java

```diff
@@ -26,9 +38,12 @@ public void shouldTransmitIstioHeadersInResponseContext() {
         requestHeaders.put(IstioHeaders.X_B3_SPANID, null);
         when(requestContext.getHeaders()).thenReturn(requestHeaders);
         when(responseContext.getHeaders()).thenReturn(responseHeaders);
-        new IstioHeaderMapperFilter().filter(requestContext, responseContext);
+        doReturn("this isn't being tested").when(istioHeaderStorage).toString();
```

Comment:
I'm not sure I understand why this is here.

Reply:
It's actually redundant now.  At some point in a previous iteration, there was a null pointer exception that, that stopped.  It's not the case any more so I've taken it out.  Good spot.